search.php

```diff
@@ -76,32 +113,44 @@
 		'name'    => '#CampaignName3',
 		'results' => 0,
 	],
+	[
+		'name'    => '#CampaignName4',
+		'results' => 0,
+	],
 ];
 $context['categories'] = [
 	[
-		'name'    => '#1',
+		'name'    => 'Organisation',
+		'results' => 0,
+	],
+	[
+		'name'    => 'Press Release',
 		'results' => 0,
 	],
 	[
-		'name'    => '#2',
+		'name'    => 'Publication',
```

Comment:
Publication and story are not categories, but are controlled by the extra taxonomy.